app/views/layouts/_nav_gamer.html.erb

```diff
@@ -7,17 +8,18 @@
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>
             </button>
-            <div id="logo-container">
-                <a class="navbar-left" href="<%= root_path %>">
-                    <%= image_tag "logo.png" %>
-                </a>
-            </div>
+            <a class="navbar-brand"" href="<%= root_path %>">
+                <%= image_tag "logo-2.png", :class => "logo-nav" %>
```

Comment:
Please use non-hashrocket syntax whenever possible so it should be `class: "logo-nav"`